lms/static/scripts/file_picker_v2/components/ErrorDisplay.js

```diff
@@ -0,0 +1,81 @@
+import propTypes from 'prop-types';
+import { createElement } from 'preact';
+
+function emailLink({ address, subject = '', body = '' }) {
+  return `mailto:${address}?subject=${encodeURIComponent(
+    subject
+  )}&body=${encodeURIComponent(body)}`;
+}
+
+/**
+ * Displays details of an error, such as a failed API call and provide the user
+ * with information on how to get help with it.
+ */
+export default function ErrorDisplay({ message, error }) {
+  let details = '';
+  try {
+    if (error.details) {
+      details = JSON.stringify(error.details, null, 2 /* indent */);
+    }
+  } catch (e) {
+    // Ignored
```

Comment:
AFAIK the only way `JSON.stringify` can fail is if the input object contains circular references, which should never happen here. Since this error comes from an arbitrary source, I've added the `try ... catch` just to be defensive.